tests/integration/targets/win_dhcp_server/tasks/tests.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Great to see the tests but you aren't really doing anything with them. You should be checking the return values, i.e. it reported a change (or not for idempotency) and checking the actual DHCP server to see if it added the scopes/done the action you specified.

Reply:
I will add some test behaviour
should I add also the  version_added: '1.4.0'  to the documentation?